webgpu.h

```diff
@@ -1295,77 +1373,200 @@ typedef struct WGPUStorageTextureBindingLayout {
     WGPUTextureViewDimension viewDimension;
 } WGPUStorageTextureBindingLayout WGPU_STRUCTURE_ATTRIBUTE;
 
+/**
+ * Filled by `::wgpuSurfaceGetCapabilities` with what's supported for `::wgpuSurfaceConfigure` for a pair of @ref WGPUSurface and @ref WGPUAdapter.
+ */
 typedef struct WGPUSurfaceCapabilities {
     WGPUChainedStructOut * nextInChain;
+    /**
+     * The bit set of supported @ref WGPUTextureUsage bits.
+     * Guaranteed to contain @ref WGPUTextureUsage_RenderAttachment.
+     */
     WGPUTextureUsageFlags usages;
+    /**
+     * A list of supported @ref WGPUTextureFormat values, in order of preference.
```

Comment:
Should we be explicit and say the first one is the preferred format?

Reply:
That's what "in order of preference" meant in my mind ^^